src/main.py

```diff
@@ -119,21 +132,17 @@ def _provide_fake_simulator_dependencies(i: DependencyContainer) -> None:
     i.provide('web_browser_client', WebBrowserClient)
     i.provide('replay_saver', ReplaySaver)
     i.provide('can_adapter', LoopbackCANAdapter)
-    i.provide('match_action_controller', MatchActionController)
-    i.provide('protobuf_handler', ProtobufHandler)
```

Comment:
Why not have them in simulation mode ?

Reply:
I moved them to the 'common' function, so they are instantiated both for simulation and production mode